Training/MainWindow.xaml.cs

```diff
@@ -100,7 +100,10 @@ private void GetLengthOfString_Click(object sender, RoutedEventArgs e)
 
         private void GetSquareOfNumber_Click(object sender, RoutedEventArgs e)
         {
-            MathsFunctions.GetSquare();
+            int first = (Int32.TryParse(Input141.Text, out first)) ? first : 0;
+            int second = (Int32.TryParse(Input142.Text, out second)) ? second : 0;
+            int numberToSquare = first + second;
+            Output141.Content = MathsFunctions.GetSquare(numberToSquare);
         }
```

Comment:
wjy we get 2 inputs for doing Square?

Reply:
because of the gui, the math function it with one input.
User can add value if each cell and get the square, if setting both cells - user will get the sum square.

Reply:
just remove the extra input from UI to have only one